engine/src/flutter/shell/platform/embedder/embedder.h

```diff
@@ -3089,6 +3113,22 @@ FlutterEngineResult FlutterEngineDispatchSemanticsAction(
     const uint8_t* data,
     size_t data_length);
 
+//------------------------------------------------------------------------------
+/// @brief      Dispatch a semantics action to the specified semantics node
+///             within a specific view.
+///
+/// @param[in]  engine  A running engine instance.
+/// @param[in]  info    The dispatch semantics on view arguments.
+///                     This can be deallocated once |FlutterEngineAddView|
+///                     returns.
+///
+/// @return     The result of the call.
+///
+FLUTTER_EXPORT
+FlutterEngineResult FlutterEngineDispatchSemanticsActionOnView(
```

Comment:
What do you think of calling this `FlutterEngineSendSemanticsAction`? 

```suggestion
FlutterEngineResult FlutterEngineSendSemanticsAction(
```